pkg/reconciler/ingress/resources/gateway.go

```diff
@@ -72,13 +72,17 @@ func GetHTTPServer(gateway *v1alpha3.Gateway) *v1alpha3.Server {
 }
 
 func belongsToClusterIngress(server *v1alpha3.Server, ia v1alpha1.IngressAccessor) bool {
-	// The format of the portName should be "<clusteringress-name>:<number>".
-	// For example, route-test:0.
+	// The format of the portName should be "<namespace>-<ingress_name>:<number>".
```

Comment:
why not separate with '/'?

Reply:
Thank you. Yes, I confirmed that it works with `/` and updated.